### PR TITLE
Update Team Members List

### PR DESCRIPTION
**Pull request by WebReflection**
Using this script on [the following page](https://github.com/orgs/nodejs/teams/modules/members),
helps members page to be sync each update.
```js
// per each page
const members = JSON.parse(localStorage.getItem('members') || '[]');
document.querySelectorAll('a.f4').forEach(a => {
  const name = a.textContent.trim();
  const slug = a.nextElementSibling ?
                a.nextElementSibling.textContent.trim() : name;
  const href = a.href;
  members.push({name, slug, href});
});
localStorage.setItem('members', JSON.stringify(members));

// sort them out after all pages
members.sort((a, b) => {
  const aSlug = a.slug.toLowerCase();
  const bSlug = b.slug.toLowerCase();
  return aSlug < bSlug ? -1 : 1;
});

// generate markdown
console.log(members.map(
  m => `* [${m.name}](${m.href}) @${m.slug}`
).join('\n'));

localStorage.removeItem('members');
```

### CODE REVIEW

**Comment by targos** [comment]
Can you please sort the list by GitHub username?

**Comment by WebReflection** [comment]
I haven't touched the list as returned by the linked page.

should it be sorted by github name instead of first name or maybe surname? I don#t know, which is why I haven't really touched the list.

TL;DR I don't mind being the one before the last one as long as everyone agrees it should be sorted by github name.

**Comment by targos** [comment]
We usually sort by GitHub name in other repos (it's easier when people can have non-ASCII characters in their real names):
https://github.com/nodejs/node#tsc-technical-steering-committee
https://github.com/nodejs/security-wg#current-project-team-members

**Comment by WebReflection** [comment]
done

**Comment by MylesBorins** [comment]
@WebReflection there are a few more members since your initial PR. The open call closes in 3 hours, might make sense to wait for that to close before updating 😄

**Comment by devsnek** [comment]
```js
Array.from(document.querySelectorAll('a.f4'))
  .map(a => ({
    name: a.textContent.trim(),
    slug: a.nextElementSibling ?
      a.nextElementSibling.textContent.trim() : name,
    href: a.href,
  }))
  .sort((a, b) => a.slug.localeCompare(b))
  .map((m) => `* [${m.name}](${m.href}) @${m.slug}`)
  .join('\n');

**Comment by WebReflection** [comment]
@devsnek nah, that doesn't work. The members page has more than one list, it's 30 per pages, so you need to store, retrieve, add, sort, print, and clean.

**Comment by devsnek** [comment]
rip 😄

**Comment by WebReflection** [comment]
@devsnek also the only reason your script didn't fail is that the global window has a name property otherwise it would've thrown a reference error ... look closer
```js
{
    name: a.textContent.trim(),
    slug: a.nextElementSibling ?
      a.nextElementSibling.textContent.trim() : name, // <-- nope
    href: a.href,
  }
```

no reason to waste time to improve such basic one-off utility but thanks for trying

**Comment by WebReflection** [comment]
@MylesBorins let me know when it's done and I'll squash the update 👍

**Comment by MylesBorins** [comment]
@WebReflection there are 3 people missing who have not yet accepted the invitation to the group

Torgny Bjers @tbjers
Ahmad Abdul-Aziz @devamaz
Matt DuLeone @mduleone

**Comment by WebReflection** [comment]
I'd like to use same script to put them in the list so ... I'll wait to squash updates.

**Comment by MylesBorins** [comment]
@WebReflection they have been invited to the org but have not accepted invitations. Might not be worth blocking this PR on them accepting

**Comment by WebReflection** [comment]
added 3 of them manually (boooooooriiiiiiiiing!!!) :smile:

**Comment by tbjers** [comment]
For some reason I didn't get the invitation, I've been actively looking for it.

**Comment by MylesBorins** [comment]
@tbjers I've resent the invite. It should go to the email address associated with your github account. Please lmk if you still have issues
